app/views/admin/definition_relations/edit.html.erb

```diff
@@ -1,7 +1,9 @@
 <% if parent?
      add_breadcrumb_item feature_link(object.child_node.feature)
-     add_breadcrumb_item link_to 'definitions', admin_feature_definitions_path(object.child_node.feature)
-     add_breadcrumb_item ts('new.this')
+     add_breadcrumb_item link_to Definition.model_name.human(count: :many).titleize.s, admin_feature_path(object.feature, section: "definitions")
```

Comment:
I think pointing to  feature_link(object.child_node.feature) and admin_feature_path(object.feature, section: "definitions") would be redundant. I would just leave first one as feature_link(object.child_node.feature, section: 'definitions').